js/script.js

```diff
@@ -151,3 +151,14 @@ const removeBook = () => {
 document.addEventListener('click', () => {
   removeBook();
 });
```

Comment:
- [ ] The remove button does not function as intended. I tried to remove a book by clicking on the remove icon, but the book was not removed from the UI and the local storage. Please work on that.